src/app/components/shared/footer/footer.component.scss

```diff
@@ -1,67 +1,73 @@
- .footer {
-  max-width: 100%;
+* {
+	margin: 0;
+	padding: 0;
+  margin-top: 6px;
+}
+
+.footer {
   background-color: #787878;
+  width: 100%;
+}
 
- }
-  h3 {
-    margin-bottom: 24px;
-  }
-  
-  .container {
-    box-sizing: border-box;
-    margin: auto;
-    padding: 0;
-    max-width: 1170px;
-    min-width: 300px;
-    font-weight: normal;
-    display: flex;
-    justify-content: center;
-  }
-  
-  .footer--blocks_contacts {
-    margin-top: 22px;
-  }
-  
-  .footer-icons {
-    margin-top: 20px;
-  }
-  
-  .pi-envelope {
-    margin-right: 11px;
+
+
+h3 {
+  margin-bottom: 24px;
+}
+
+.container {
+  box-sizing: border-box;
+  margin: auto;
+  padding: 0;
+  max-width: 1170px !important;
```

Comment:
Please avoid using !important as much as you can

Reply:
Fixed